contrib/fwupd.spec.in

```diff
@@ -122,6 +122,15 @@ BuildArch: noarch
 %description tests
 Data files for installed tests.
 
+%package agent
+Summary: Agent to upload local device list
+Requires: %{name}%{?_isa} = %{version}-%{release}
+
+%description agent
+An agent that runs once per day to upload the local device list to a
+central server. Most users will not need this, and this is only useful
+in some enterprise deployments.
+
```

Comment:
Any particular reason to keep this in a separate package for Fedora?  Since daemon, command line frontend and tool are all in the same package why split this one out?

Reply:
I didn't want an executable running on millions of computers every day, needlessly. This agent functionality is really designed for enterprise deployments that deploy a custom image, rather than just installing from a LiveUSB. I also didn't want to deal with the privacy trolls telling me fwupd is doing more evil things.

Reply:
Well flip side is if it's on millions of computers already but not running it's way easier to set up.
Just "install" it in the same standard packages but have the systemd service that uses it default to disabled.

Reply:
Part of the _initial enrollment process_ could even be enabling that systemd service too.

Reply:
Would that be StartUnit, or something like EnableUnitFiles?

Reply:
I think `EnableUnitFiles`

Reply:
I've added a FIXME, if you want to have a go with `EnableUnitFiles` please be my guest :)

Reply:
I've added this, seems to work.